src/app/course-item/course-item.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+import { Course } from '../course';
+
+
+@Component({
+  selector: 'app-course-item',
+  templateUrl: './course-item.component.html',
+  styleUrls: ['./course-item.component.scss']
+})
+
+export class CourseItemComponent implements OnInit {
+
+  @Input() course: Course;
+  
+  @Output() delete = new EventEmitter<number>();
+
+  constructor() { }
+
+  ngOnInit() {
+  }
+
+  deleteItem(id: number) {
```

Comment:
You have a discrepancy. In the template, this method is called `delete`. As a result, this method is never called